CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Next Version
+### Breaking Changes
+- Upgrade to signers 1.0.18 removes support for deprecated SECP256K1 curve.
```

Comment:
suggestion: Upgrade to signers 1.0.18 removes support for deprecated SECP256K1 curve in Azure remote signing.